commit d9cf99ad7ea14d1c17c2d034d89d97854e889643
Author: Jon Anderson <jon.anderson@joyent.com>
Date:   2019-03-12T14:52:05-04:00 (7 months ago)
    
    MANTA-4165 Remove diffutils package pre-requisite from Muppet readme
    Reviewed by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
    Approved by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>

diff --git a/README.md b/README.md
index 2ca0b1e..e31fb65 100644
--- a/README.md
+++ b/README.md
@@ -28,16 +28,14 @@ Run `make prepush` before commits; otherwise, follow the
 
 ## Prerequisites
 
-To properly run the muppet tests, two additional packages need to be installed
-from pkgsrc.
+To properly run the muppet tests, the following package must be installed from
+pkgsrc:
 
-  -  haproxy
-  -  diffutils
+- haproxy
 
 Though the version of haproxy might differ from the running `loadbalancer` zone,
 it is simply used to check that haproxy can properly parse the resulting
-`haproxy.cfg` files generated in the tests.  Additionally, `GNU diff` is used to
-compare the generated files with an expected output.
+`haproxy.cfg` files generated in the tests.
 
 ## Running the Tests
 
